app/src/main/res/xml/locales_config.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<locale-config xmlns:android="http://schemas.android.com/apk/res/android">
+    <locale android:name="en"/>
+    <locale android:name="de"/>
+    <locale android:name="it"/>
+    <locale android:name="ko"/>
+    <locale android:name="nl"/>
+    <locale android:name="pt-rBR"/>
```

Comment:
pt-rBR correct?

Reply:
Android studio shows these abbreviations in the android project sidebar, which is where I got that from
![image](https://github.com/dessalines/jerboa/assets/1341797/01fc4230-9faf-4b1c-b0a3-eb4ffe9c6936)

You're right though, its supposed to just be `pt-BR`.